tests/constraints/tests.py

```diff
@@ -80,21 +83,38 @@ def test_repr(self):
     def test_deconstruction(self):
         fields = ['foo', 'bar']
         name = 'unique_fields'
-        check = models.UniqueConstraint(fields=fields, name=name)
-        path, args, kwargs = check.deconstruct()
+        constraint = models.UniqueConstraint(fields=fields, name=name)
+        path, args, kwargs = constraint.deconstruct()
         self.assertEqual(path, 'django.db.models.UniqueConstraint')
         self.assertEqual(args, ())
         self.assertEqual(kwargs, {'fields': tuple(fields), 'name': name})
 
+    def test_deconstruction_with_condition(self):
+        fields = ['foo', 'bar']
+        name = 'unique_fields'
+        constraint = models.UniqueConstraint(fields=fields, name=name, condition=models.Q(foo=models.F('bar')))
```

Comment:
Did you omit a variable for `condition` (unlike `fields` and `name`) for some reason?

Reply:
no reason, add `condition` variable like `fields` and `name`